start.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 #memcached -d
-gunicorn main:app -w 4 --reload --timeout=200 --threads=4 -k uvicorn.workers.UvicornWorker --bind 0.0.0.0:5000
\ No newline at end of file
+gunicorn main:app -w 4 --reload --timeout=200 --threads=4 -k uvicorn.workers.UvicornWorker --bind 0.0.0.0:5010
```

Comment:
why change to port 5010?
we cant keep that for the deployment to the cluster. Ingress is listening on port 5000

Reply:
Ah, ok. I am using my setting. I need this one for my machine. What would be the best way to be synch with your branch?